### PR TITLE
feat(behavior_velocity_planner): use XXXStamped in autoware_internal_debug_msgs

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

Run the following command, and manually add `autoware_internal_debug_msgs` in the package.xml
```
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/string_stamped/autoware_internal_debug_msgs\/msg\/string_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::StringStamped/autoware_internal_debug_msgs::msg::StringStamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/float64_stamped/autoware_internal_debug_msgs\/msg\/float64_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::Float64Stamped/autoware_internal_debug_msgs::msg::Float64Stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/float32_stamped/autoware_internal_debug_msgs\/msg\/float32_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::Float32Stamped/autoware_internal_debug_msgs::msg::Float32Stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/int32_stamped/autoware_internal_debug_msgs\/msg\/int32_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::Int32Stamped/autoware_internal_debug_msgs::msg::Int32Stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/float64_multi_array_stamped/autoware_internal_debug_msgs\/msg\/float64_multi_array_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::Float64MultiArrayStamped/autoware_internal_debug_msgs::msg::Float64MultiArrayStamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs\/msg\/float32_multi_array_stamped/autoware_internal_debug_msgs\/msg\/float32_multi_array_stamped/g' {} \;
find ./ -type f -exec sed -i -e 's/tier4_debug_msgs::msg::Float32MultiArrayStamped/autoware_internal_debug_msgs::msg::Float32MultiArrayStamped/g' {} \;
```

## Related links

https://github.com/autowarefoundation/autoware/issues/5580

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by soblin — APPROVED** [review]

**Comment by takayuki5168** [comment]
@yuki-takagi-66 @maxime-clem 
FYI:
For the maintainers of the crosswalk and run_out modules, I will merge this PR once the CI passes.

**Pull request merged**
